src/Faker/Provider/pl_PL/LicensePlate.php

```diff
@@ -529,14 +530,14 @@ public static function licensePlate(
     /**
      * Selects random area from the list of available and requested.
      */
-    protected static function selectRandomArea(array $available, ?array $requested)
+    protected static function selectRandomArea(array $available, ?array $requested): string
```

Comment:
@scybulski 

Can you remove the return type declaration here to avoid a backwards-incompatible change?

Reply:
PHP 7.4 which is minimum for this project supports method type hints. So what is it meant to be backwards-compatible with?

Reply:
Nevermind, I've just seen the bc job output.